app/models/papi_client.rb

```diff
@@ -327,6 +328,9 @@ def get_command_line(study_file:, action:, user_metrics_uuid:, params_object: ni
     when 'image_pipeline'
       # image_pipeline is node-based, so python command line to this point no longer applies
       command_line = 'node expression-scatter-plots.js'
+    when 'ingest_anndata'
+      # extract cluster data from AnnData file, currently hardcoding the obsm-keys
+      command_line +=  " --ingest-anndata --anndata-file #{study_file.gs_url} --extract-cluster --obsm-keys ['X_umap','X_tsne']"
```

Comment:
This would be a bit more coherent to put above `ingest_subsample`.  It's upstream of that in our dataflow and quite similar to the ingest jobs for our core study file types, whereas e.g. `differential_expression` and `image_pipeline` are fairly different from a dataflow / product perspective.